components/MegaMenu.vue

```diff
@@ -28,8 +17,8 @@
 <script lang="tsx">
 import { SfMegaMenu, SfList, SfMenuItem } from "@storefront-ui/vue"
 import { ref, onMounted, defineComponent, computed } from "@vue/composition-api"
-import { useCategoryTree, useUiHelpers } from "@/composables"
-import { categoryGetters } from "@/composables/getters"
+import { useCategoryTree, useUiHelpers } from "../composables"
```

Comment:
Switch back to using alias "@/composables"

Reply:
using '@' marking with red underline by vetur saying -- Cannot find module '@/composables/getters' or its corresponding type declarations but it's working . So reverted the changes to '@'.